test/integration/targets/docker_swarm/tasks/main.yml

```diff
@@ -13,11 +13,16 @@
     diff: no
     ignore_errors: yes
 
+  - name: Kill docker daemon
+    command: systemctl kill -s 9 docker
+    become: yes
+
   - name: Restart docker daemon
     service:
       name: docker
-      state: restarted
+      state: started
```

Comment:
One last try: can you change this back to `restarted`? I somehow have the feeling that `systemctl kill` doesn't put the unit into `stopped` state, so `state: started` doesn't do anything.

Reply:
Done. Maybe I'm missing something but a lot of failures have been in the task before these (named Cleanup (trying)). This is where some CI timeouts have occurred. Maybe one option could be to use `command` with `timeout 1m docker swarm leave -f`?

Reply:
Interesting. I wasn't aware of that task taking forever, I thought it fails a lot (that's why it got `ignore_errors`), but I wasn't aware that it hits the 45 min timeout. Do you know where that happened?

From the runs I found in this PR (https://app.shippable.com/github/ansible/ansible/runs/116966/59/console, https://app.shippable.com/github/ansible/ansible/runs/116906/59/console, https://app.shippable.com/github/ansible/ansible/runs/116901/59/console, https://app.shippable.com/github/ansible/ansible/runs/116898/59/console, https://app.shippable.com/github/ansible/ansible/runs/116838/58/console, https://app.shippable.com/github/ansible/ansible/runs/116835/59/console), the problem was always the `Restart docker daemon` task.

Reply:
Yes you're right. I missed that CI continued on to run `Restart docker daemon` after it logged an error message in `Cleanup (trying)`.

Reply:
Can you trigger another CI run, so we can see whether the new cleanup works more reliably?

Reply:
Can you try another time? Just so it hopefully works three times in a row ;)